<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />

<meta name="description" content="The Shen Lab - Laboratory of Yufeng Shen, Ph.D. - Center for Computational Biology and Bioinformatics, Department of Systems Biology, and Department of Biomedical Informatics, Columbia University - Research focus: DNA sequencing, genome assembly computational genomics, statistical genetics, pharmacogenomics" />
<meta name="keywords" content="Shen, Columbia University, Computational Genomics, Bioinformatics, genomics, genome, sequencing, pharmacogenomics" />

<title>Shen Lab | Research</title>
<link rel="stylesheet" type="text/css" href="web.css">
</head>

<body>

<div id="container">
  <div id="header">    
   
        
      <h1>Shen Lab of Computational Genomics</h1>



    <div id="menu">
      <ul>
       <li><a href="index.html">Home</a></li>
      <li><a href="people.html">People</a></li>
    <li class="active"><a href="research.html">Research</a></li>
      <li><a href="papers.html">Papers</a></li>
      <li><a href="software.html">Software & Data</a></li>
            <li><a href="teaching.html">Teaching</a></li>
      <li><a href="contact.html">Contact</a></li>
    
      </ul>
      <div></div>
    </div>
  </div>

  <div id="mainContent">

<center>
<a href="#genetics">Human Genetics</a> &nbsp; | &nbsp; <a href="#genomics">Genome Sequencing</a> &nbsp; | &nbsp; <a href="#immunogenomics">Immunogenomics</a> </center>



<div class="project">

<a name="genetics"></a>
<h3>Human Genetics</h3>
<p>Identify genetic causes of human diseases by integrative analysis of genetics variants with functional genomics rooted in biological mechanisms. To improve the ability to identify risk variants and genes, we develop new computational methods to integrate gene expression and epigenomic data with genetic data. These methods are usually based on machine learning models with heuristics developed from biological mechanisms, such as haploinsufficiency. We apply these methods in genetic studies of human diseases in collaboration with Dr. Wendy Chung's group. In the last few years, we have been focusing on developmental disorders, such as congenital diaphragmatic hernia, congenital heart disease, and autism.</p>
<h4>Recent papers</h4>
 
 <ol>
 <li><strong>Han X*, Chen S*, Flynn E, Wu S, Wintner D, Shen Y</strong>, (2018) Distinct epigenomic patterns are associated with haploinsufficiency and predict risk genes of developmental disorders, <em>bioRxiv</em> <a href="https://www.biorxiv.org/content/early/2018/04/02/205849">205849</a>. to be published at <em>Nature Communications</em></li>
 
 <li>SPARK Consortium &, (2018) SPARK: A US Cohort of 50,000 Families to Accelerate Autism Research, <em>Neuron</em>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/29420931">PMID: 29420931 </a></li>

<li><strong>Zhu N</strong>*, Gonzaga-Jauregui C*, Welch CL*, Ma L, Qi H, King AK, Krishnan U, Rosenzweig EB, Ivy DD, Austin ED, Hamid R, Nichols WC, Pauciulo MW, Lutz KA, Sawle A, Reid JG, Overton JD, Baras A, Dewey F, <strong>Shen Y</strong>, Chung WK. (2018) Exome Sequencing in Children With Pulmonary Arterial Hypertension Demonstrates Differences Compared With Adults. <em>Circulation: Genomic and Precision Medicine</em>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/29631995">PMID: 29631995 </a></li>

<li><strong>Qi H</strong>*, Yu L*, <strong>Zhou X*, Kitaygorodsky A</strong>,   Wynn J, <strong>Zhu N</strong>, Aspelund G, Lim YF, Crombleholme T, Cusick R, Azarow K, Danko ME, Chung D, Warner BW, Mychaliska GB, Potoka D, Wagner AJ, ElFiky M, Nickerson DA, Bamshad MJ, Wilson JM, High FA, Longoni M, Donahoe P, Chung WK#, <strong>Shen Y</strong>#, (2017) Genetic analysis of <em>de novo</em> variants reveals sex differences in complex and isolated congenital diaphragmatic hernia and indicates <em>MYRF</em> as a candidate gene, <em>bioRxiv</em>, <a href="https://www.biorxiv.org/content/early/2017/11/12/206037">206037</a></li>


<li>Zhang C, <strong>Shen Y</strong>, (2017) A cell type-specific expression signature predicts haploinsufficient autism-susceptibility genes. <em>Human Mutation</em>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/27860035">PMID: 27860035</a></li>


<li><strong>Qi H</strong>, Dong C, Chung WK, Wang K, <strong>Shen Y</strong>, (2016) Deep Genetic Connection between Cancer and Developmental Disorders. <em>Human Mutation</em>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/27363847">PMID:  27363847 </a></li>

<li>Homsy J*, Zaidi S*, <strong>Shen Y</strong>*, Ware JS, Samocha KE, Karczewski KJ, DePalma SR, McKean D, Wakimoto H, Gorham J, Jin SC, Deanfield J, Giardini A, Porter GA Jr, Kim R, Bilguvar K, López-Giráldez F, Tikhonova I, Mane S, Romano-Adesman A, Qi H, Vardarajan B, Ma L, Daly M, Roberts AE, Russell MW, Mital S, Newburger JW, Gaynor JW, Breitbart RE, Iossifov I, Ronemus M, Sanders SJ, Kaltman JR, Seidman JG, Brueckner M, Gelb BD, Goldmuntz E, Lifton RP, Seidman CE, Chung WK. (2015) De novo mutations in congenital heart disease with neurodevelopmental and other congenital anomalies. <em>Science</em>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/26785492">PMID: 26785492</a></li>

<li>Yu L, Sawle AD, Wynn J, Aspelund G, Stolar CJ, Arkovitz MS, Potoka D, Azarow KS, Mychaliska GB, <strong>Shen Y</strong>#, Chung WK#, (2015) Increased burden of <em>de novo</em> predicted deleterious variants in complex congenital diaphragmatic hernia, <em>Human Molecular Genetics</em>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/26034137">PMID: 26034137</a></li>

</ol>


<center>* * * * * * *</center>


<a name="genomics"></a>
<h3>Genome sequencing</h3></a>
<p>Develop computational methods to identify and interpret genetic variations from genome sequencing data. Ongoing projects include detecting copy number variants and structural variants from exome or genome sequencing data, accurate estimation of genotype likelihood from sequencing data using deep learning, predicting genetic effect of missense or noncoding variants. </p>

<h4>Recent papers</h4>
 
 <ol>
 <li><strong>Qi H*, Chen C*, Zhang H, Long JJ</strong>, Chung WK, Guan Y, <strong>Shen Y</strong>. (2018) MVP: predicting pathogenicity of missense variants by deep learning. <em>bioRxiv</em>, <a href="https://www.biorxiv.org/content/early/2018/04/02/259390">259390</a></li>
 
 
 <li><strong>Backenroth D</strong>, Homsy J, Murillo LR, Glessner J, <strong>Lin E</strong>, Brueckner M, Lifton R, Goldmuntz E, Chung WK, <strong>Shen Y</strong>, (2014) CANOES: Detecting rare copy number variants from whole exome sequencing data, <strong><em>Nucleic Acids Research</em></strong> doi: 10.1093/nar/gku345  <a href="http://www.ncbi.nlm.nih.gov/pubmed/24771342">PMID: 24771342</a></li>
 
 <li><strong>Shen Y</strong>#, Song R, and Pe'er I#, (2011) Coverage Tradeoffs and Power Estimation in the Design of Whole-Genome Sequencing Experiments for Detecting Association, <em>Bioinformatics</em>, PMID: 21636589</li>
 
  <li><strong>Shen Y</strong>#, Wan Z, Coarfa C, Drabek R, Chen L, Ostrowski EA, Liu Y, Weinstock GM, Wheeler DA, Gibbs GA, Yu F#, A SNP discovery method to assess variant allele probability from next generation resequencing data. <em>Genome Research</em> 2010, 20(2):273-80. Epub 2009 Dec 17.</li>
 
<li><strong>Shen Y</strong>, Sarin S, Liu Y, Hobert O, Pe'er I, Comparing Platforms for C. elegans Mutant Identification Using High-Throughput Whole-Genome Sequencing. <em>PLoS ONE</em> 2008, 3(12): e4012</li>

<li>Wheeler DA*, Srinivasan M*, Egholm M*, <strong>Shen Y</strong>*, et al,  Weinstock GM, Gibbs RA, Rothberg JM, (2008) The complete genome of an individual by massively parallel DNA sequencing, <em>Nature</em>, PMID: 18421352 </li>
	
 
</ol>

<center>* * * * * * *</center>


<a name="immunogenomics"></a>
<h3>Immunogenomics</h3></a>
<p>Computational analysis and mathematical modeling to understand dynamics of immune cells in human.</p>

<h4>Recent papers</h4>


<ol>
<li>Lee J*, Zhou YJ*, <strong>Ma W*</strong>, Zhang W, Aljoufi A, Luh T, Lucero K, Liang D, Thomsen M, Bhagat G, <strong>Shen Y</strong> #, Liu K #, (2017) Lineage specification of human dendritic cells is marked by IRF8 expression in hematopoietic stem cells and multipotent progenitors. <em>Nature Immunology</em>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/28650480">PMID: 28650480</a></li>

<li>Kumar BV*, <strong>Ma W</strong>*, Miron M, Granot T, Guyer RS, Carpenter DJ, Senda T, Sun X, Ho SH, Lerner H, Friedman AL, <strong>Shen Y</strong> #, Farber DL #, (2017) Human Tissue-Resident Memory T Cells Are Defined by Core Transcriptional and Functional Signatures in Lymphoid and Mucosal Sites, <em>Cell Reports</em>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/28930685">PMID: 28930685</a></li>

<li>Sims JS, <strong>Grinshpun B</strong>, <strong>Feng Y</strong>, Ung TH, Neira JA, Samanamud JL, Canoll P, <strong>Shen Y</strong> #, Sims PA #, Bruce JN#, (2016) Diversity and divergence of the glioma-infiltrating T-cell receptor repertoire. <em>PNAS</em>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/27261081"> PMID: 27261081 </a></li>

<li>DeWolf S, <strong>Shen Y</strong>, Sykes M, (2016)  A New Window into the Human Alloresponse. <em>Transplantation</em>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/26760572">PMID: 26760572</a></li>

<li>Morris H, DeWolf S, Robins H, Sprangers B, LoCascio SA, Shonts BA, Kawai T, Wong W, Yang S, Zuber J, <strong>Shen Y</strong> #, Sykes M #, (2015) Tracking donor-reactive T cells: Evidence for clonal deletion in tolerant kidney transplant patients, <em>Science Translational Medicine</em>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/25632034">PMID: 25632034</a></li>

<li>Thome JJC, Yudanin N, Ohmura Y, Kubota M, <strong>Grinshpun B</strong>, Sathaliyawala T, Kat T, Lerner H, <strong>Shen Y</strong>, Farber DL, (2014) Spatial Map of Human T Cell Compartmentalization and Maintenance over Decades of Life, <strong><em>Cell</em></strong>, DOI: <a href="http://dx.doi.org/10.1016/j.cell.2014.10.026">http://dx.doi.org/10.1016/j.cell.2014.10.026</a></li>

<li><strong>Grinshpun B</strong>, Sims J, Canoll P, Bruce JN, Sims P, <strong>Shen Y</strong>, (2013) Analyzing T cell repertoire diversity by high-throughput sequencing, <em>Global Conference on Signal and Information Processing (GlobalSIP), 2013 IEEE</em>, doi: 10.1109/GlobalSIP.2013.6736810 </li>


</ol>



</div>     

  </div>
  
     <div class='cleaner'> <p>&nbsp; </p> </div>


  <div id="footer">
      <p>  <center> </center></p>
  </div>
  
</div>

<script src="http://www.google-analytics.com/urchin.js" type="text/javascript">
</script>
<script type="text/javascript">
_uacct = "UA-400081-5";
urchinTracker();
</script>

</body>
</html>
