<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />

<meta name="description" content="The Shen Lab - Laboratory of Yufeng Shen, Ph.D. - Department of Systems Biology, and Department of Biomedical Informatics, Columbia University - Research areas: genome sequencing, computational genomics, human genetics, immunogenomics" />
<meta name="keywords" content="Shen, Columbia University, Computational Genomics, Bioinformatics, Human Genetics, Sequencing" />

<title>Shen Lab | Software and Data</title>
<link rel="stylesheet" type="text/css" href="web.css">
</head>

<body>

<div id="container">
  <div id="header">    
  
        
        
      <h1>Shen Lab of Computational Genomics</h1>



    <div id="menu">
      <ul>

 <li><a href="index.html">Home</a></li>
      <li><a href="people.html">People</a></li>
      <li><a href="research.html">Research</a></li>
      <li><a href="papers.html">Papers</a></li>
      <li class="active"><a href="software.html">Software & Data</a></li>
            <li><a href="teaching.html">Teaching</a></li>
      <li><a href="contact.html">Contact</a></li>            

      </ul>
      <div></div>
    </div>
  </div>

  <div id="mainContent">

      <h2>Software</h2>

<ul>
<li><a href="https://github.com/ShenLab/episcore">Episcore</a> (Updated 2018), a method to predict gene haploinsufficiency based on human epigenomic profiles under normal conditions, as described in <a href="https://www.biorxiv.org/content/early/2018/04/02/205849">Han, Chen et al 2018</a></li> 
<li><a href="https://github.com/ShenLab/missense">MVP</a> (updated 2018), a method to predict deleterious genetic effect of missense variants, as described in <a href="https://www.biorxiv.org/content/early/2018/04/02/259390">Qi et al 2018</a>.</li>

<li><a href="https://github.com/ShenLab/hotspot">HotSpots</a> (Updated 2016) a method to infer cancer somatic mutation hotspots, as descriibed in <a href="https://www.ncbi.nlm.nih.gov/pubmed/27363847">Qi et al 2016</a></li>
<li><a href="canoes/">CANOES</a>, (Updated 2014) a method to call CNVs from exome sequencing data with arbitrary number of reference samples.</li>
<li><a href="https://github.com/ShenLab/Repertoire">Repertoire</a>, (Updated 2015) Scripts for analyzing T cell receptor repertoire sequencing data.</li>
<li><a href="http://genomecenter.columbia.edu/ngs/software/opera/index.php">OPERA </a>, (updated 2012) an online power calculator for genome sequencing and genetic studies</li>
<li><a href="zinfandel/">Zinfandel</a>, (updated 2012) calling Copy Number Variants (CNVs) from whole genome data based on both depth of coverage and mate pair information</li>

 <li><a href="http://code.google.com/p/atlas-snp/">Atlas-SNP</a>, (last updated 2010) bioinformatics analysis of next-generation sequencing, optimized for variant calling from 454 data</li>
<li><a href="http://code.google.com/p/genometer/">Genometer</a>, (last updated 2011) estimating genome size in the presence of repeats</li>


 </ul>


<h2>Data</h2>

<h4>Homsy et al, Science, 2015: <em>de novo</em> mutations from 1120 congenital heart disease cases</h4>
<p><a href="http://science.sciencemag.org/highwire/filestream/646355/field_highwire_adjunct_files/1/aac9396_SupportingFile_Other_seq1_v4.zip">All supplementary tables (a zip file)</a>:
<ol>
<li>S1: Phenotypes for each case proband, including cardiac, neurodevelopmental disorders and extra-cardiac congenital anomalies.</li>
<li>S2: List of de novo Mutations in CHD case cohort.</li>
<li>S3: List of de novo Mutations in Control cohort. </li>
<li>S4: List of de novo probabilities for each variant class in each protein-coding gene on the Nimblegen V2 exome, adjusted for depth in Cases.</li>
<li>S5: List of de novo probabilities for each variant class in each protein-coding gene on the Nimblegen V2 exome, adjusted for depth in Controls. </li>
<li>S6: Functional term enrichment analysis of all Genes with Damaging (loss of function + deleterious missense) de novo mutations in all cases. </li>
<li>S7: Functional term enrichment analysis of all Genes with Loss of Function de novo mutations in 860 new cases.</li>
<li>S8: List of 1,563 variants (1,161 unique genes) with damaging de novo mutations from 7 independent NDD cohorts.</li>
<li>S9: Functional term enrichment analysis among 69 genes with Damaging de novo mutations overlapping between CHD cases and the published NDD (P-NDD) cohort. </li>
<li>S10: Percentile ranks of genes by expression in the developing mouse heart and brain. </li>
</ol>

</p>

<h4>GWAS of drug adverse reactions</h4>
<p><em>GWAS</em> data sets from Serious Adverse Event Consortium, related to these papers (<a href="http://www.ncbi.nlm.nih.gov/pubmed/19483685">Daly et al 2009</a>; <a href="http://www.ncbi.nlm.nih.gov/pubmed/21221126">Shen et al 2011</a>; <a href="http://www.ncbi.nlm.nih.gov/pubmed/21221126">Lucena et al 2011</a>; Overby et al 2014) can be found at <a href="https://dataportal.saeconsortium.org">SAEC Data Portal</a> (Registration required). Processed files ready for PLINK are available upon request (&#121;&#115;&#104;&#101;&#110;&#64;&#99;&#50;&#98;&#50;&#46;&#99;&#111;&#108;&#117;&#109;&#98;&#105;&#97;&#46;&#101;&#100;&#117;). 
</p>

<h4>Detecting CNVs from exome sequencing</h4>
<p><em>Exome sequencing</em> and genotyping data used in CANOES (<a href="http://www.ncbi.nlm.nih.gov/pubmed/24771342">Backenroth et al 2014</a>) is from NHLBI Pediatric Cardiac Genomics Consortium (PCGC) and available through <a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000571.v1.p1">dbGaP</a>
</p>



  </div>
   <div id="footer">
     <p> &nbsp; </p>
  </div>
  
  
</div>

<script src="http://www.google-analytics.com/urchin.js" type="text/javascript">
</script>
<script type="text/javascript">
_uacct = "UA-400081-5";
urchinTracker();
</script>

</body>
</html>
